homeassistant/components/ista_ecotrend/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration registers no actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: Has coordinator for centralized data retrieval. No need for base entity as there is only one platform.
+  config-flow-test-coverage: done
```

Comment:
Reauth tests are missing the abort if we try to setup a different account during reauth